games-action/supertuxkart/metadata.xml

```diff
@@ -5,6 +5,9 @@
 		<email>hasufell@posteo.de</email>
 		<name>Julian Ospald</name>
 	</maintainer>
+	<maintainer>
+		<email>nado@troglodyte.be</email>
+		<name>Corentin Pazdera</name>
```

Comment:
here was the closing tag missing, I fixed it in 48d0a4c0a4eff183fbe339262ef5207051efe7e8